src/mumble/UserLocalVolumeSlider.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright 2005-2020 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "UserLocalVolumeSlider.h"
+
+#include <cmath>
+
+// We define a global macro called 'g'. This can lead to issues when included code uses 'g' as a type or parameter name
+// (like protobuf 3.7 does). As such, for now, we have to make this our last include.
+#include "Global.h"
+#include <QtWidgets/qgridlayout.h>
+#include <QtWidgets/qtooltip.h>
```

Comment:
```suggestion
#include <QtWidgets/qgridlayout.h>
#include <QtWidgets/qtooltip.h>
```

The Global class does not seem to be needed